utils/utils.go

```diff
@@ -16,31 +16,51 @@ import (
 
 var Emoji = "\U0001F430" + " Keploy:"
 var ConfigGuide = `
-  # Example on using globalNoise
-  # globalNoise:
-  #    global:
-  #      body: {
-  #         # to ignore some values for a field,
-  #         # pass regex patterns to the corresponding array value
-  #         "url": ["https?://\S+", "http://\S+"],
-  #      }
-  #      header: {
-  #         # to ignore the entire field, pass an empty array
-  #         "Date": [],
-  #       }
-  #     # to ignore fields or the corresponding values for a specific test-set,
-  #     # pass the test-set-name as a key to the "test-sets" object and
-  #     # populate the corresponding "body" and "header" objects
-  #     test-sets:
-  #       test-set-1:
-  #         body: {
-  #           # ignore all the values for the "url" field
-  #           "url": []
-  #         }
-  #         header: {
-  #           # we can also pass the exact value to ignore for a field
-  #           "User-Agent": ["PostmanRuntime/7.34.0"]
-  #         }
+# Example on using tests
```

Comment:
Please make an another variable to this because this configGuide is specific to global noise and this  text is used for error purpose in global noise case I guess.

Reply:
This is for the generate config command. Incase the record command fails due to decoding of yaml file this would be printed. It isn't specific to global noises.

<img width="987" alt="image" src="https://github.com/keploy/keploy/assets/60891544/c18c18ac-96dc-46b9-b941-aa7568895fbe">
As mentioned this would log incase any thing fails with YAML, a generic and not a specific case